### PR TITLE
Added blog about flash removal

### PR DESCRIPTION
**Pull request by mrocajr**

### CODE REVIEW

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+author:
+  name: Michael Roca
+  github: mrocajr
+date: 2016-06-01 14:50:06
+---
+
+In August of 2016, we [announced] our intention of removing flash as a part of the core video.js project. 
+As html5 video becomes the standard playback tech and flash fades into obsolescence, it is time 
+to remove flash from the core player and move it to a separate code base.  This will give us the ability 
+to allow developer to continue to support legacy browsers by adding the tech themselves, while allowing
```
Comment (jakemingolla) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: developer -> developers

Comment (mrocajr) [reply source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: Thanks:)

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+date: 2017-01-25 14:50:06
+---
+
+In August of 2016, we [announced] our intention of removing flash as a part of the core video.js project.
+As html5 video becomes the standard playback tech and flash fades into obsolescence, it is time 
+to remove flash from the core player and move it to a separate code base.  This will give us the ability 
+to allow developers to continue to support legacy browsers by adding the tech themselves, while allowing 
+us to minimize legacy code in video.js and decrease the footprint of the player.
+
+This follows in the footsteps of Chrome, Safari and Firefox which are all taking steps to depricate flash.
```
Comment (brandonocasey) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: ...steps to ~~depricate~~ deprecate flash...

**Review by brandonocasey — COMMENTED** [review]
Two comment LGTM otherwise

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+---
+title: Video.js removes flash from core player
+tags:
+  - videojs 6
```
Comment (brandonocasey) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: should be `video.js 6.0` for consistency

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md (hunk 2 above)
Comment (mrocajr) [reply source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: Thanks!  Fixed it.

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+title: Video.js removes flash from core player
+tags:
+  - videojs 6.0
```
Comment (brandonocasey) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: needs to `video.js 6.0` with the `.` between `video` and `js`

Comment (mrocajr) [reply source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: I'll get it right one of these days:)

**Review by brandonocasey — APPROVED** [review]

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  - video.js 6.0
+  - flash
+  - html5
+author:
+  name: Michael Roca
+  github: mrocajr
+date: 2017-01-25 14:50:06
+---
+
+In August of 2016, we [announced] our intention of removing flash as a part of the core video.js project.
```
Comment (mister-ben) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: Flash and Video.js should be capitalised throughout as proper nouns.

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+---
+title: Video.js removes flash from core player
+tags:
+  - Video.js 6.0
```
Comment (gkatsev) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: tag is all lowercase: `video.js 6.0`

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  - Flash
+  - html5
+author:
+  name: Michael Roca
+  github: mrocajr
+date: 2017-01-25 14:50:06
+---
+
+In August of 2016, we [announced] our intention of removing flash as a part of the core video.js project.
```
Comment (gkatsev) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: needs to be `[announced][]` to use the link below.

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+As html5 video becomes the standard playback tech and flash fades into obsolescence, it is time 
+to remove flash from the core player and move it to a separate code base.  This will give us the ability 
+to allow developers to continue to support legacy browsers by adding the tech themselves, while allowing 
+us to minimize legacy code in video.js and decrease the footprint of the player.
+
+This follows in the footsteps of Chrome, Safari and Firefox which are all taking steps to deprecate flash.
+
+Chrome: [Chrome 56 disables flash by default]
```
Comment (gkatsev) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: needs the extra brackets here and in the rest of the links to be linked properly

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+In the meantime, the separate [videojs-flash] project has been created for Flash tech support.
+
+```html
+<link rel="stylesheet" href="path/video.js/dist/video-js.css">
+<script src="path/video.js/dist/video.js"></script>
+<script src="path/videojs-flash/dist/videojs-flash.js"></script>
+<video  id='vid' class='video-js' controls height=300 width=600>
+<source src="video.mp4" type="video/mp4">
+</video>
+<script> var player = videojs('vid', {techOrder: ['flash']}); </script>
```
Comment (gkatsev) [inline source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: flash is added to the tech order automatically, might be worth mentioning.

**Review by gkatsev — CHANGES_REQUESTED** [review]

File: source/_drafts/Video.js-Flash-Removed-From-Core-Player.md (hunk 6 above)
Comment (mrocajr) [reply source/_drafts/Video.js-Flash-Removed-From-Core-Player.md]: All set.

**Review by gkatsev — APPROVED** [review]

**Pull request merged**